phonon/model.py

```diff
@@ -0,0 +1,35 @@
+import phonon.fields
+import phonon.exceptions
+
+
+class MetaModel(type):
+    def __new__(cls, name, parents, dct):
+        cls._fields = {}
+        for key, val in dct.items():
+            if isinstance(val, phonon.fields.Field):
+                cls._fields[key] = val
```

Comment:
The `MetaModel` sets all the fields in a `_fields` `dict` similar to how `django` does for it's models.